src/main/kotlin/com/craftschemers/hub/tasks/StartGameTask.kt

```diff
@@ -0,0 +1,42 @@
+package com.craftschemers.hub.tasks
+
+import com.craftschemers.hub.Hub
+import com.craftschemers.hub.minigame.AbstractGameData
+import com.craftschemers.hub.minigame.Minigame
+import org.bukkit.Bukkit
+import org.bukkit.ChatColor
+
+class StartGameTask(private val gameData: AbstractGameData, private val game: Minigame) : Runnable {
+
+    private var time = 10
+    private var id = -1
```

Comment:
Instead of a placeholder value this could be nullable
```suggestion
    private var id: Int? = null
```